### DISCUSSION TITLE
Unable to take documents photo by BACK camera for educational document proof

### DISCUSSION THREAD

**Discussion by IAmTahazzot** [community/community#63716] [GitHub Education]
### Hi everyone,

In github Upload proof of your academic status page, I can't take photo by my back camera.

There are two option while taking a photo,

Camera 2 1, facing front ✅ Works
Camera 2 0, facing back ❌ Doesn't work

It's nearly impossible to take documents photo with facing front camera. Switching to Camera 2 0, facing back doesn't work, it just stay in front camera. I don't see other way to submit form.

Is that any specific device issue? or website problem? I'm using Vivo Y20G (Android 13)

**Comment by safnimj** [comment]
Capture the two side of your ID by your normal camera and merge it together and upload it..
Only the image type will accepted

**Reply by Starmania** [reply]
Uploading image lower your chances of pass, because the photo was edited. Also, one side card (even without date) can pass

**Comment by 190-785** [comment]
> ### Hi everyone,
> In github Upload proof of your academic status page, I can't take photo by my back camera.
> 
> There are two option while taking a photo,
> 
> Camera 2 1, facing front ✅ Works Camera 2 0, facing back ❌ Doesn't work
> 
> It's nearly impossible to take documents photo with facing front camera. Switching to Camera 2 0, facing back doesn't work, it just stay in front camera. I don't see other way to submit form.
> 
> Is that any specific device issue? or website problem? I'm using Vivo Y20G (Android 13)

Yeah, Same thing is happening to me too, can anyone provide us with a solution as to how we can fix this issue as it is basically impossible to take photos of a document through a front facing camera.

**Reply by Starmania** [reply]
Trial and error... Only way

**Reply by 190-785** [reply]
> Trial and error... Only way

Any way to upload images? no option regarding it was showing.

**Reply by Ayush-Kumar-web** [reply]
@Starmania  I have tried several times each time they ask for different proof even after canceling the current request

**Comment by PengJiahao7890** [comment]
I have the same problem, sad.

**Reply by 190-785** [reply]
Hi, I actually found a solution to it, you can use Iruin app to use your phone as a virtual camera and then you would be able to use backcamera of your phone through your laptop.
https://play.google.com/store/apps/details?id=com.jacksoftw.webcam

**Comment by 190-785** [comment]
Hi, I found a solution to it, You can use the Iruin app to use your phone as a virtual camera and then you would be able to use the backcamera of your phone through your laptop.
You might have to use split screen in laptop or if you have another device it can also be used.
https://play.google.com/store/apps/details?id=com.jacksoftw.webcam

**Comment by RonaldDPR** [comment]
Hi, Someone found a solution? I have the same problem.Try the recommendation of the irunesas webcam app. But when I enter the website de Hithub, it only recognizes my laptop's webcam.

**Comment by alpert13** [comment]
I find a temporarely "fix".
I use the split window feature of my phone, i open the browser and the camera app.
First, make sure that the back camera is selected in the camera app. Then switch to browser and choose the back camera on github page. Now the back camera is selected.
Sorry if my english is not good

**Comment by h1ac** [comment]
I used the opera browser and It let me choose the back camera! I got accepted :)

**Comment by Cl9s4bl3** [comment]
Hi! I've got a working with a bit of a trick. (I used Firefox)

First, I selected what type of document I want to approve.
Then I took a selfie (doesn't really matter what's on it) and then BEFORE I pressed on "Reset", I chose the back camera from the list below. After this, I pressed on "Reset" and It worked, and I could take the picture with the back camera. 2 minutes later, I've got approved.
